implementations/micrometer-registry-appoptics/src/main/java/io/micrometer/appoptics/AppOpticsMeterRegistry.java

```diff
@@ -96,24 +96,27 @@ public void start(ThreadFactory threadFactory) {
     protected void publish() {
         try {
             for (List<Meter> batch : MeterPartition.partition(this, config.batchSize())) {
+                String body = batch.stream()
+                        .map(meter -> meter.match(
+                                this::writeGauge,
+                                this::writeCounter,
+                                this::writeTimer,
+                                this::writeSummary,
+                                this::writeLongTaskTimer,
+                                this::writeTimeGauge,
+                                this::writeFunctionCounter,
+                                this::writeFunctionTimer,
+                                this::writeMeter)
+                        )
+                        .filter(Optional::isPresent)
+                        .map(Optional::get)
+                        .collect(joining(",", "{\"measurements\":[", "]}"));
+                if (body.equals("{\"measurements\":[]}")) {
```

Comment:
@izeye  Thanks for jumping on this so quick!

does it make sense to have a Static String for this?  

```
private Static final String EMPTY_MEASURES_BODY = "{\"measurements\":[]}"; 
...
if (body.equals(EMPTY_MEASURES_BODY)) {

Reply:
@CandiedCode Thanks for the feedback! I updated in d9377b8 as suggested.

Reply:
From the bytecode perspective, it is a constant string either way. If we want to try to make sure we don't update one part and not the other (resulting in a bug), it'd be better to make the prefix and suffix constants and then use the concatenation of those in the `equals` method.

Reply:
@shakuzen Sounds good to me. I pushed d90696f.